js/src/wcdl/section/card/index.js

```diff
@@ -9,8 +9,8 @@ import { Card as WPCard } from '@wordpress/components';
 import Body from './body';
 import Footer from './footer';
 
-const Card = ( props ) => {
-	return <WPCard { ...props } size="none" />;
+const Card = ( { size = '', ...props } ) => {
```

Comment:
📝 
`'none'` is not a valid size value, so changed it to an empty string to avoid appearing `is-size-none` CSS class in the card nodes.

![2022-03-31 16 42 45](https://user-images.githubusercontent.com/17420811/161015624-c4c36e82-7e43-414b-9943-2e4f56c6898e.png)